### PR TITLE
Add PAF Page

### PR DESCRIPTION
**Pull request by ghickman**
This adds the Project Application Form edit page, updates the Project with changes from it, implements the show more toggle, and toggles the checkmark in the Support Docs.

Ref #1330

### CODE REVIEW

File: opentech/apply/projects/views.py [hunk 1]
```diff
@@ -49,3 +50,20 @@ class ApplicantProjectDetailView(DetailView):
...
     applicant_view = ApplicantProjectDetailView
+
+
+@method_decorator(staff_required, name='dispatch')
+class ProjectEditView(UpdateView):
+    form_class = ProjectEditForm
+    model = Project
+
+    def form_valid(self, form):
+        self.object = form.save(commit=False)
```
Comment (todd-dembrey) [inline opentech/apply/projects/views.py]: I like to do the following to stay out the way of inheritance.
```python      
if form.has_changed():
    form.instance.user_has_updated_details = True
return super().form_valid(form)
````
I honestly dont know which is the "correct" approach, but personally dislike re-writing the `form_valid` method this way. 

Alternatively should this be on the form save method?

File: opentech/apply/projects/templates/application_projects/project_detail.html [hunk 2]
```diff
@@ -101,10 +101,19 @@ <h5>Email</h5>
                         -
                         {% endif %}
                     </div>
+                </div>
+
+                <a class="link--reveal-proposal js-toggle-propsoal-info" href="#">Show more</a>
```
[the remark this answers is not in the document]
Comment (ghickman) [reply opentech/apply/projects/templates/application_projects/project_detail.html]: Fixed in 63bd1de8

File: opentech/apply/projects/views.py [hunk 3]
```diff
@@ -49,3 +50,20 @@ class ApplicantProjectDetailView(DetailView):
...
+
+        if form.has_changed():
+            self.object.user_has_updated_details = True
+
+        print(self.object.user_has_updated_details)
```
[the remark this answers is not in the document]
Comment (ghickman) [reply opentech/apply/projects/views.py]: Fixed and rebased.

File: opentech/apply/projects/views.py (hunk 1 above)
Comment (ghickman) [reply opentech/apply/projects/views.py]: Fixed and rebased.

File: opentech/apply/projects/forms.py [hunk 4]
```diff
@@ -24,6 +25,35 @@ def save(self, *args, **kwargs):
...
+    class Meta:
+        fields = [
+            'title',
+            'contact_legal_name',
+            'contact_email',
+            'contact_address',
+            'contact_phone',
+            'value',
+            'proposed_start',
+            'proposed_end',
```
Comment (todd-dembrey) [inline opentech/apply/projects/forms.py]: Would it be worth some logic to confirm end > start?

File: opentech/apply/projects/models.py [hunk 5]
```diff
@@ -11,10 +12,16 @@ class Project(models.Model):
...
     title = models.TextField()
 
-    contact_legal_name = models.TextField(default='')
-    contact_email = models.TextField(default='')
-    contact_address = models.TextField(default='')
+    contact_legal_name = models.TextField(_('Person or Organisation name'), default='')
+    contact_email = models.TextField(_('Email'), default='')
+    contact_address = models.TextField(_('Address'), default='')
+    contact_phone = models.TextField(_('Phone'), default='')
     value = models.DecimalField(default=0, max_digits=10, decimal_places=2)
```
Comment (todd-dembrey) [inline opentech/apply/projects/models.py]: Do we want a min number validator on this? Should have been on the model definition PR but was thinking about it when reviewing the form.

`validators=[MinValueValidator(Decimal('0.01'))`

https://docs.djangoproject.com/en/dev/ref/validators/#minvaluevalidator

Comment (ghickman) [reply opentech/apply/projects/models.py]: Fixed in dc518330

File: opentech/apply/projects/forms.py (hunk 4 above)
Comment (ghickman) [reply opentech/apply/projects/forms.py]: Fixed in ad8154a

**Pull request merged**